services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/attachment/AttachmentTestUtil.kt

```diff
@@ -26,6 +26,7 @@ val DUMMY_DATA = "ABC".toByteArray()
 val DEFAULT_DATA by lazy {
     "/fi/hel/haitaton/hanke/decision/fake-decision.pdf".getResourceAsBytes()
 }
+const val DEFAULT_SIZE = 18959L
```

Comment:
Could this be `DEFAULT_DATA.len()`? To explain, where the magic number comes from.

Reply:
Now that I moved fake-decision.pdf under test (was under itest) this could be done.  Before moving it there were errors in unit tests that used DEFAULT_SIZE (mainly attachment Factory companion object initialization).